components/dropdown/test/dropdown-menu.visual-diff.js

```diff
@@ -1,27 +1,40 @@
-const puppeteer = require('puppeteer');
-const VisualDiff = require('@brightspace-ui/visual-diff');
-const helper = require('./dropdown-helper.js');
-
-describe('d2l-dropdown-menu', () => {
-
-	const visualDiff = new VisualDiff('dropdown-menu', __dirname);
-
-	let browser, page;
-
-	before(async() => {
-		browser = await puppeteer.launch();
-		page = await browser.newPage();
-		await page.setViewport({ width: 800, height: 800, deviceScaleFactor: 2 });
-		await page.goto(`${visualDiff.getBaseUrl()}/components/dropdown/test/dropdown-menu.visual-diff.html`, { waitUntil: ['networkidle0', 'load'] });
-		await page.bringToFront();
-	});
-
-	after(() => browser.close());
-
-	it('first-page', async function() {
-		await helper.open(page, '#dropdown-menu');
-		const rect = await helper.getRect(page, '#dropdown-menu');
-		await visualDiff.screenshotAndCompare(page, this.test.fullTitle(), { clip: rect });
-	});
-
-});

```

Comment:
Is your IDE switching CRLF/LF's around?

Reply:
I didn't even see that - weird. But it's only this one file?

Reply:
Yeah I dunno. VS Code actually has a little thing in the status bar that shows you which it's using for a particular file, and clicking it lets you switch it back and forth.

Reply:
TIL: The VScode line-endings button exists.
This was switched to LF (which is what we want) so there's no issue. Going to add linting for line-endings in another PR.